### ISSUE TITLE
The right-clicked menu bar is not displayed correctly on Runtime Identifier "win-x64".

### ISSUE THREAD

**Issue by ych98** [OutSystems/WebView#333]
![image](https://github.com/OutSystems/WebView/assets/104015241/a4541745-d6a1-4afd-95a5-80c0bea62cee)

**Comment by joaompneves** [comment]
Is this your app or the Sample App?

**Comment by ych98** [comment]
This is true for both programs

**Comment by joaompneves** [comment]
On my M1 (apple) everything seems ok running this code: https://github.com/OutSystems/WebView/tree/avalonia11/SampleWebView.Avalonia

Don't have windows, so I can't help.

**Comment by ych98** [comment]
I tried CefGlue.Demo.Avalonia and found that it has the same problem with win-x64.
![1709258571900_C3B2B5EC-6D73-41cc-BF36-0B9F41545272](https://github.com/OutSystems/WebView/assets/104015241/1d6704b8-85be-4256-8d35-9b575de98cf2)
At the same time, I found out what the problem was.
When I build on mac, the Resource folder is automatically generated and contains resource files such as locale.pak, while when I build on windows, there is no Resource folder, only the locales folder and the content is empty.
![image](https://github.com/OutSystems/WebView/assets/104015241/06a49b67-d93a-41d4-9731-3210b780e656)
![image](https://github.com/OutSystems/WebView/assets/104015241/87520d96-0596-45fc-bd5e-4e6921a3ea37)
It should be that the context menu is not displayed properly due to a resource loading issue.
